contracts/ethregistrar/ETHRegistrarController.sol

```diff
@@ -1,178 +1,299 @@
 pragma solidity >=0.8.4;
 
-import "./PriceOracle.sol";
 import "./BaseRegistrarImplementation.sol";
 import "./StringUtils.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
 import "../resolvers/Resolver.sol";
+import "../registry/ReverseRegistrar.sol";
+import "../dnssec-oracle/BytesUtils.sol";
+import "./IETHRegistrarController.sol";
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@ensdomains/name-wrapper/interfaces/INameWrapper.sol";
```

Comment:
Why not use https://github.com/ensdomains/ens-contracts/blob/master/contracts/wrapper/INameWrapper.sol instead?